deployment/ansible/roles/cac-tripplanner.app/meta/main.yml

```diff
@@ -3,7 +3,6 @@
     - { role: "azavea.git" }
     - { role: "azavea.nginx" }
     - { role: "azavea.nodejs", nodejs_npm_version: "2.1.5" }
-    - { role: "azavea.packer" }
+    - { role: "azavea.packer", packer_version: "0.7.5" }
```

Comment:
It can sometimes be harder to find these when things go wrong. I like the style of this better, but in practice it had led to longer debugging cycles.